packages/frontend/components/lobby/onlinePlayers.tsx

```diff
@@ -104,6 +103,7 @@ const OnlinePlayers = (props: Props): ReactElement => {
           .online_players_inner_wrapper :global(tbody) {
             display: block;
             overflow: auto;
+            height: 300px;
```

Comment:
I'll need to wait till I have a safari-capable device to test this. But this line bothers me as it should be dynamically set, not hard coded. Does this element properly fill the rest of the screen with this?